src/styles/components/_card.scss

```diff
@@ -0,0 +1,164 @@
+@import "../abstracts/_variables.scss";
+@import "../base/base";
+
+@media only screen and (max-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(1, 1fr);
+    max-width: $card-width;
```

Comment:
don't set max width calculated by hand here, let the content dictate how much space should be for this container.